src/config.js

```diff
@@ -279,9 +276,11 @@ export default class Config {
       });
 
       this.registries[key] = registry;
-      this.registryFolders.push(registry.folder);
+      if (this.registryFolders.indexOf(registry.folder) === -1) {
```

Comment:
this fixes an issue i was seeing where value of `registryFolders` was `['node_modules', 'node_modules']` - which I think was causing double lookups.